stubs/flake8-bugbear/METADATA.toml

```
@@ -0,0 +1 @@
+version = "21.11.*"
```

As `flake8-bugbear` uses calver, this should be an exact version number:

```suggestion
version = "21.11.29"
```